components/map/map.jsx

```diff
@@ -124,6 +152,55 @@ export default class Map extends React.Component {
         this.props.onPokemonClick(randomPokemon);
       }
     });
+
+    setTimeout(() => this.updatePokemonPosition(pokemonMarker, map), 1);
```

Comment:
Because if you don't have it, the pokemon would be too easy to catch since the user can just do a double click since the pokemon will always be in the center of the map before the first position change can even occur.